lib/v1/gadgets/blogger.md

```diff
@@ -0,0 +1,12 @@
+gratipay.xml is a an example of Gadget for Blogger. To add it, go to
+you blog overview, then:
+
+    Layout -> Add a Gadget -> Add your own
+
+Enter URL ('https://' prefix didn't work for me):
+
+    grtp.co/v1/gadgets/blogger.xml
```

Comment:
Are gadgets specific to blogger or will we have more of them? If specific to blogger I suggest:

`grtp.co/v1/blogger.xml`